app/main/presenters/search_summary.py

```diff
@@ -31,7 +24,16 @@ def write_list_as_sentence(input_list, final_conjunction):
             formatted_conjunction = " {} ".format(final_conjunction)
             return formatted_conjunction.join([u', '.join(start), end])
 
-    def __init__(self, results_total, request_args, filter_groups, lots_by_slug):
+    def __init__(self, results_total, request_args, filter_groups, lots_by_slug, markup_tags=True):
+        self.markup_tags = markup_tags
+
+        self.COUNT_PRE_TAG = '<span class="search-summary-count">' if markup_tags else ''
+        self.COUNT_POST_TAG = '</span>' if markup_tags else ''
+        self.KEYWORDS_PRE_TAG = '<em>' if markup_tags else ''
+        self.KEYWORDS_POST_TAG = '</em>' if markup_tags else ''
+        self.LOT_PRE_TAG = '<em>' if markup_tags else ''
+        self.LOT_POST_TAG = '</em>' if markup_tags else ''
```

Comment:
This lot feel like they break DRY. To be honest I think I'd have preferred an approach that stripped the tags off when we wanted a non-HTML version (e.g. using `lxml`), rather than having to pepper the existing code with tests.

Reply:
Currently we only have lxml as a dev dependency. Do we really want to include that in the main app?

Reply:
Personally I'd be fine with that but I guess you'd better ask around. And if other people disagree that this would be better then by all means ignore me on this.

Reply:
Done. 👍